geanygendoc/docs/Makefile.am

```diff
@@ -15,6 +15,6 @@ dist_pluginhtmldoc_DATA = \
 if BUILD_RST
 manual.html: manual.rst manual.css html4css1.css
 	$(AM_V_GEN) $(RST2HTML) -d --strict \
-		--stylesheet-path $(abs_srcdir)/html4css1.css,$(abs_srcdir)/manual.css \
+		--stylesheet-path $(srcdir)/html4css1.css,$(srcdir)/manual.css \
```

Comment:
Using `abs_srcdir` was done on purpose, see de89404b0edec7cc6a9808e94dd2559ebabdbe10.
If somehow this breaks MSYS2, it can be weighted again as it's not 2016 anymore, but you'd have to verify no relevant version of the tool is still impacted by the issue.

Reply:
What does the command-line look like? would quoting the paths help?

Reply:
I reverted this change as I found that if I build from the root of the repo it works

Reply:
That's even more bizarre given they were *absolute* paths… but glad it worked.

We can revisit that in a more specific changest if it's useful, but again it'd need verifying it doesn't "unfix" something else.